docs/source/common/optimizers.rst

```diff
@@ -397,6 +397,8 @@ For example, here step optimizer A every batch and optimizer B every 2 batches.
         if optimizer_idx == 1:
             if (batch_idx + 1) % 2 == 0:
                 optimizer.step(closure=optimizer_closure)
+            else:
+                # closure/training_step won't be called in this case
```

Comment:
```suggestion
                ...  # closure/training_step won't be called in this case
```

Reply:
Instead of adding an `else`, why not add `closure/training_step would only be called in this case` comment under if statement?

Reply:
no specific reason, one or the other way is equivalent in the information it provides.
doing both is probably redundant